run_kraken.py

```diff
@@ -37,9 +38,32 @@ def main(cfg):
     logging.info("Starting kraken")
 
     # Parse and read the config
-    if os.path.isfile(cfg):
-        with open(cfg, "r") as f:
-            config = yaml.full_load(f)
+    # Initializing found_config variable as False to test config is a file or not
+    found_config = False
+
+    try:
+        # For config as local file
+        if os.path.isfile(cfg):
+            with open(cfg, "r") as f:
+                config = yaml.full_load(f)
+            # Set found_config as True since config is local file
+            found_config = True
+
+        # For config as remote file
+        else:
+            content = requests.get(cfg)
+            if content.status_code == 200:
+                texts = content.text
+                config = yaml.full_load(texts)
+                # Set found_config as True since config is remote file
+                found_config = True
+
```

Comment:
think if you add an else statement here that throws an exception, all cases will be covered that you don't need to have the found_config variable